api/envoy/extensions/filters/udp/dns_filter/v3alpha/dns_filter.proto

```diff
@@ -52,8 +53,26 @@ message DnsFilterConfig {
     google.protobuf.Duration resolver_timeout = 1 [(validate.rules).duration = {gte {seconds: 1}}];
 
     // DNS resolution configuration which includes the underlying dns resolver addresses and options.
+    // *dns_resolution_config* will be deprecated once
+    // :ref:`typed_dns_resolver_config <envoy_v3_api_field_extensions.filters.udp.dns_filter.v3alpha.DnsFilterConfig.ClientContextConfig.typed_dns_resolver_config>`
+    // is fully supported.
     config.core.v3.DnsResolutionConfig dns_resolution_config = 2;
 
+    // DNS resolver type configuration extension. This extension can be used to configure c-ares, apple,
+    // or any other DNS resolver types and the related parameters.
+    // For example, an object of
+    // :ref:`CaresDnsResolverConfig <envoy_v3_api_msg_extensions.network.dns_resolver.cares.v3.CaresDnsResolverConfig>`
+    // can be packed into this *typed_dns_resolver_config*. This configuration will replace the
+    // :ref:`dns_resolution_config <envoy_v3_api_field_extensions.filters.udp.dns_filter.v3alpha.DnsFilterConfig.ClientContextConfig.dns_resolution_config>`
+    // configuration eventually.
+    // TODO(yanjunxiang): Investigate the deprecation plan for *dns_resolution_config*.
```

Comment:
s/Investigate/Determine/

also, nit: add extra line above the TODO.

Reply:
This also needs to be `[#comment:TODO .... ]` so it does not leak into generated documentation. Please check other please in .proto files for TODOs and wrap then into [#comment: ].

Reply:
ACK